testsuite/MDAnalysisTests/converters/test_openmm_parser.py

```diff
@@ -101,6 +102,25 @@ def test_segids(self, top):
         else:
             assert top.segids.values == []
 
+    def test_elements(self, top):
+        if 'elements' in self.expected_attrs:
+            assert len(top.elements.values) == self.expected_n_atoms
+            assert isinstance(top.elements.values, np.ndarray)
+            assert all(isinstance(elem, str) for elem in top.elements.values)
```

Comment:
Hmm, I suppose I'm not sure how much this kind of test adds. The resname/resnum/segids test if there's an empty list or an array; a corresponding `else` here would be helpful too. Especially for the some/no elements subclasses below.

Reply:
I'm not able to figure out what the corresponding else should have, since elements is not expected as an attribute, what am I supposed to test? Do you mean to say I should test if there's an empty list or array like the tests you mentioned. Also, I modified the tests for types and masses to check for an empty array like resname/resnum etc, I hope it's fine.

Reply:
What if the OpenMM input has no elements?

Reply:
> What if the OpenMM input has no elements?

Umm in that case the the parsed topology too shouldn't have elements, is that right?

Reply:
Yes -- and it would be good to have a test for this :)

Reply:
Okay this was so obvious :), anyway thanks for the hint. Will add the tests